resources/assets/js/videos/videoContainer.vue

```diff
@@ -102,6 +102,8 @@ export default {
             hasCrossOriginError: false,
             videoFilenames: null,
             focusInputFindlabel: false,
+            corsRequestBreaksVideo: false,
+            showErrorMsg: true,
```

Comment:
I was confused what this variable actually does. It is called `showErrorMsg` but actually the intention is to *hide* the error messages while the "load with CORS" attempt is running. Maybe rename the variable (e.g. `attemptWithCors`)?